src/main/java/org/commonwl/view/util/FileUtils.java

```diff
@@ -47,6 +47,34 @@ public static void deleteGitRepository(Git repo) throws IOException {
     }
   }
 
+  /**
+   * Deletes the directory of a temporary git repository. Note that the <code>Git</code> object contains a
+   * repository with a directory, but this directory points to the
+   *
+   * <pre>.git</pre>
+   *
+   * directory. This method will delete the parent of the
+   *
+   * <pre>.git</pre>
+   *
+   * directory, which corresponds to the cloned folder with the source code from git.
+   * Since temporary folders are generated using <code>UUID.randomUUID()</code> instead of the commit hex digest,
+   * if the fodler name contains a '-' character it is identified as temporary.
```

Comment:
s/fodler/folder

Reply:
I added some new test and fixed the existing ones